cmd/common.go

```diff
@@ -1,111 +0,0 @@
-package cmd
```

Comment:
use git move for this. This seems like a new file created in profiles

Reply:
I did use git mv for this , but once i add the changes required to this file, it ends up marking this as a new file. It only happens for this files and works for the rest.